hammr/commands/image/image.py

```diff
@@ -197,6 +198,99 @@ def help_publish(self):
         doParser = self.arg_publish()
         doParser.print_help()
 
+    def arg_deploy(self):
+        doParser = ArgumentParser(prog=self.cmd_name + " deploy", add_help=True,
+                                  description="Deploy an instance of a published image on the targeted cloud.")
+        mandatory = doParser.add_argument_group("mandatory arguments")
+        mandatory.add_argument('--id', dest='pid', required=True,
+                               help="the ID of the published image to deploy")
+        mandatory.add_argument('-n', '--name', dest='deploy_name', required=True,
+                               help="the name of the image to deploy")
+
+        optional = doParser.add_argument_group("optional arguments")
+        optional.add_argument('--vcpu', dest='vcpu', required=False,
```

Comment:
Can you add the default value in the help ?
Maybe something like this "if none provided, assume 1"